lightning/src/ln/chan_utils.rs

```diff
@@ -217,87 +217,100 @@ pub fn derive_public_key<T: secp256k1::Signing>(secp_ctx: &Secp256k1<T>, per_com
 
 /// Derives a per-commitment-transaction revocation key from its constituent parts.
 ///
+/// Only the transaction broadcaster owns a valid witness to propagate a revoked
+/// commitment transaction, thus per_commitment_secret always come from broadcaster
```

Comment:
nit: s/come/comes/, twice plus in comments below.

Reply:
Not gonna to take it, to avoid modifying commits